src/constants/events.ts

```diff
@@ -3,4 +3,6 @@ export const CONNECT_EVENT = 'CONNECT'
 export const TRANSACTION_EVENT = 'TRANSACTION_SUBMITTED'
 export const ATTRIBUTION_EVENT = 'ATTRIBUTION'
 export const REFERRER_EVENT = 'REFERRER'
+// CAUGHT events
```

Comment:
Wrongly placed comment?

Reply:
bad naming, but if we want to use single event for AUTO/non AUTO caught events then i'll remove it